Changed 1 thing in SickoMenu

HarithGamerPk opened the pull request:
The change is:
You will be able to see a confirmed label once u click on unlock all cosmetics

redphantom1000 commented:
huh

g0aty commented:
Would be better if only the required code was changed, not the entire file. Still looks good though!

g0aty commented:
Would be better if only the required code was changed, not the entire file. Still looks good though!

HarithGamerPk commented:
> Would be better if only the required code was changed, not the entire file. Still looks good though!

ill work on that today ill just modify the code dont worry

HarithGamerPk commented:
ill keep the original then modify it

> Would be better if only the required code was changed, not the entire file. Still looks good though!

ill replace it with the original and then replace the affected code

HarithGamerPk commented:
> Would be better if only the required code was changed, not the entire file. Still looks good though!

yeah i did it i replaced the older code with the source code one and changed that required part with my edited part
